src/ts/commands.ts

```diff
@@ -75,35 +75,66 @@ export const requestMatchesForDirtyRangesCommand = (
 };
 
 /**
- * Indicate new hover information is available. This could include
- * details on hover coords if available (for example, if hovering
- * over a match decoration) to allow the positioning of e.g. tooltips.
+ * Indicate the user is hovering over a match.
  */
-export const indicateHoverCommand = (
-  matchId: string,
-  hoverInfo?: IStateHoverInfo
-): Command => (state, dispatch) => {
+export const startHoverCommand = (matchId: string): Command => (
+  state,
+  dispatch
+) => {
   if (dispatch) {
     dispatch(
       state.tr.setMeta(
         PROSEMIRROR_TYPERIGHTER_ACTION,
-        newHoverIdReceived(matchId, hoverInfo)
+        newHoverIdReceived(matchId)
       )
     );
   }
   return true;
 };
 
 /**
- * Indicate that the user is no longer hovering over a
- * prosemirror-typerighter tooltip.
+ * Indicate that the user is no longer hovering over a match.
  */
 export const stopHoverCommand = (): Command => (state, dispatch) => {
   if (dispatch) {
     dispatch(
       state.tr.setMeta(
         PROSEMIRROR_TYPERIGHTER_ACTION,
-        newHoverIdReceived(undefined, undefined)
+        newHoverIdReceived(undefined)
+      )
+    );
+  }
+  return true;
+};
+
+
+/**
+ * Indicate the user is hovering over a match.
+ */
+export const startHighlightCommand = (matchId: string): Command => (
```

Comment:
Is this command is, like `startHoverCommand`, intended to indicate a user is hovering over a match? The comment says it is, but I am not clear what is the difference between the two. I think I'm lacking context for the terminology here, perhaps.

Reply:
@jennygrahamjones Brilliant spot, I'd seen this and forgotten to amend – the two states are different. Does the updated comment make more sense?